### DISCUSSION TITLE
Ti alpha phase, referenceframe error change [11-20] to [10-10]?

### DISCUSSION THREAD

**Discussion by Lu-Peter** [mtex-toolbox/mtex#2028] [Ask Anything]
Dear Mtex friends, 
Happy New Year !

I tried to plot the misorientation axis using Mtex. 3D printed Ti alloys. Dataformat, '.h5'.

The highest density axis should be [11-20], but I alway get the plots at [10-10], like the one below.
 
![image](https://github.com/mtex-toolbox/mtex/assets/152282113/e947c8d3-056d-41fe-b2c5-a7352fe7a4a7)

I tried all below when importing the data to Mtex. The results are the same.

%% Correct Data

rot = rotation.byEuler(90*degree,0*degree,0*degree);
ebsd = rotate(ebsd,rot,'keepEuler');

OR

%% Correct Data

rot = rotation.byEuler(90*degree,0*degree,0*degree);
ebsd = rotate(ebsd,rot,'keepXY');

OR
%% Correct Data

rot = rotation.byEuler(90*degree,0*degree,0*degree);
ebsd = rotate(ebsd,rot);

Would you please help to correct this issue? Thank you very much

Cheers
SL

**Comment by kilir** [comment]
Hi,
since you are inspecting misorientations, the absolute orientation (the one you are trying to correct for in approach 1 or 2) does not matter. So it should not be surprising that you do not see any effect.

Are you shure that your definition of your crystal symmetry is correct? https://mtex-toolbox.github.io/CrystalReferenceSystem.html

**Reply by Lu-Peter** [reply]
Hi Kilir

Sorry for late reply. Please see below. The definition of the crystal is right.
kind regards
SL.

%% Import Script for EBSD Data
%
% This script was automatically created by the import wizard. You should
% run the whoole script or parts of it in order to import your data. There
% is no problem in making any changes to this script.

%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = {... 
  'notIndexed',...
  crystalSymmetry('6/mmm', [3 3 4.7], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Titanium (Alpha) ', 'color', [0.53 0.81 0.98]),...
  crystalSymmetry('m-3m', [3.2 3.2 3.2], 'mineral', 'Titanium (Beta) ', 'color', [0.56 0.74 0.56])};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');

%% Specify File Names

% path to files
pname = 'G:\Research data\Ti';

% which files to be imported
fname = [pname '\Ti_03No.h5'];

%% Import the Data

% create an EBSD variable containing the data
ebsd = EBSD.load(fname,CS,'interface','h5');

%% Correct Data

rot = rotation.byEuler(90*degree,0*degree,0*degree);
ebsd = rotate(ebsd,rot,'keepXY');
